### DISCUSSION TITLE
Issues running https

### DISCUSSION THREAD

**Discussion by Azazel666** [outline/outline#2971] [Self Hosting]
I have setup a local running Outline Wiki with docker and i can access it through ip and port.
I have also setup a dns name in cloudflare and setup traefik for it in docker but when i try to browse the site i get a bunch of:
https://wiki.joraas.space/api/auth.config net::ERR_CERT_AUTHORITY_INVALID

and then i get a:
Uncaught (in promise) NetworkError {message: 'A network error occurred, try again?', name: 'NetworkError', stack: 'NetworkError: A network error occurred, try again?…ce/static/main.21f88946988016b83723.js:2:1402241)'}

i dont really know what im doing wrong here. This is my env file without auth and s3 stuff.
```
\# Copy this file to .env, remove this comment and change the keys. For development
\# with docker this should mostly work out of the box other than setting the Slack
\# keys (for auth) and the SECRET_KEY.
\# Please use `openssl rand -hex 32` to create SECRET_KEY
SECRET_KEY=581e8bf8a074a3a705eea972e3488a9db81aee6abb34d4c46027e5a##########
UTILS_SECRET=4db63d1eaf26426cf504dc280d92608197128f5804c264113ee0f9d#########


\# Must point to the publicly accessible URL for the installation
URL=https://wiki.joraas.space
PORT=3000

\# Optional. If using a Cloudfront distribution or similar the origin server
\# should be set to the same as URL.
\#CDN_URL=http://localhost

\# enforce (auto redirect to) https in production, (optional) default is true.
\# set to false if your SSL is terminated at a loadbalancer, for example
FORCE_HTTPS=false

ENABLE_UPDATES=true
DEBUG=cache,presenters,events,emails,mailer,utils,multiplayer,server,services

\#See translate.getoutline.com for a list of available language codes and their
\#percentage translated.
DEFAULT_LANGUAGE=en_US
HOST=https://wiki.joraas.space/
HTTP_PORT=3005
PGSSLMODE=disable
```

About the ports the application is running on port 3000 inside the container, and published public port for the container in docker is 3005.

**Comment by tommoor** [comment]
If your error is related to SSL (Certificate authority invalid) then it's unrelated to Outline which does not deal with SSL at all currently. I'd suggest your problem is probably in the traefik config somewhere

**Reply by Azazel666** [reply]
The browser does not complain on the certificate, it says the connection is secure. Im using traefik and letsencrypt to automatically generate certificates for my applications running in docker and i dont have issues with other applications but here i got an issue, something in this setup is complaining. i dont have any volumes configured for the outline application in docker so everything is default only the environment file settings i have to play with, but i cannot get the https to work through the cloudflare dns

**Comment by Azazel666** [comment]
I had setup a extra proxy/loadbalancer between router and server and this seams to cause issues for the outline application i guess some kind of data is not properly forwarded, i got the outline web to work through my dns name by just bypassing the loadbalancer and go straight from the router to the actual server instead.

**Reply by Bence98007** [reply]
I can see your documentation login screen, how did you manage to do the email/pass auth? Is it without google/slack auth? Or you are not using outline anymore?
Thanks for replying! 👍

**Reply by Azazel666** [reply]
By bypassing the NGINX loadbalancer i had in my network between router and my server running the docker containers i was able to get my site working through https on the cloadflare dns name. 
Regarding the login, only login i could make work was with slack, using google you seem to need a workspace which is a paid option, and same seems to be the case for azure (could not get it working with the fre Azure AD). For open id auth i have not found a way on how to set it up, tried using keycloak and auth0 bot got nowhere fast.

**Comment by heyvaldemar** [comment]
hey guys, I wrote [a detailed guide](https://www.heyvaldemar.com/install-outline-and-keycloak-using-docker-compose/) after I was able to configure Outline with MinIO and Keycloak.

Hopefully, it will help others.
